packages/elements-core/src/components/RequestSamples/convertRequestToSample.ts

```diff
@@ -1,7 +1,27 @@
+import { safeParse, safeStringify } from '@stoplight/json';
+import { HttpMethod } from '@stoplight/types';
+import { safeStringify as safeStringifyYaml } from '@stoplight/yaml';
 import { Request as HarFormatRequest } from 'har-format';
 import HTTPSnippet from 'httpsnippet';
 
+import { PartialHttpRequest } from '../MarkdownViewer/CustomComponents/CodeComponent';
+
 export const convertRequestToSample = (language: string, library: string | undefined, request: HarFormatRequest) => {
+  // Stoplight Markdown support
+  if (language === 'StoplightMarkdown') {
+    let markdown;
+    if (library === 'yaml') {
+      markdown =
+        '```yaml http\n' +
+        `${safeStringifyYaml(toPartialHttpRequest(request), { indent: 2, noRefs: true, skipInvalid: true })}` +
+        '\n```';
```

Comment:
This can be basically anything, like:
```tsx
if (library) {
  return;
}
```

Just so we don't break the `convertRequestTosample`